src/mono/mono/mini/llvm-intrinsics.h

```diff
@@ -257,6 +257,8 @@ INTRINS_OVR(WASM_ANYTRUE_V16, wasm_anytrue, Wasm, sse_i1_t)
 INTRINS_OVR(WASM_ANYTRUE_V8, wasm_anytrue, Wasm, sse_i2_t)
 INTRINS_OVR(WASM_ANYTRUE_V4, wasm_anytrue, Wasm, sse_i4_t)
 INTRINS_OVR(WASM_ANYTRUE_V2, wasm_anytrue, Wasm, sse_i8_t)
+INTRINS(WASM_SHUFFLE, wasm_shuffle, Wasm)
```

Comment:
It seems that you've remove the main logic for wasm specific intrinsics. If so, it is better to remove these as well.